scripts/tests/chiptest/__init__.py

```diff
@@ -59,7 +59,8 @@ def _GetManualTests() -> Set[ManualTest]:
     # for this reason are marked as manual".
     #
     # We are working to get this list down to 0.
-    manualtests.add(ManualTest(yaml="OTA_SuccessfulTransfer.yaml", reason="TODO"))
+    manualtests.add(ManualTest(
+        yaml="OTA_SuccessfulTransfer.yaml", reason="TODO"))
```

Comment:
This looks like dueling python formatters between you and someone else?

Reply:
Fixed. I have some local "format on save" which seems to be much more strict on line length than restyler. Looks ugly though, so reverted.